pkg/engine/ahocorasick/ahocorasickcore.go

```diff
@@ -1,9 +1,10 @@
-package engine
+package ahocorasick
```

Comment:
This had to be moved out of the engine package, per https://github.com/trufflesecurity/trufflehog/pull/2264#discussion_r1437829049.